### DISCUSSION TITLE
When easyeffects starts, all input devices volumes are set to 100%

### DISCUSSION THREAD

**Discussion by foodornt** [wwmm/easyeffects#1890] [Q&A]
Title

how do I override this behavior?

**Comment by wwmm** [comment]
As EasyEffects does not try to set device volume what you are seeing is probably a bug in PipeWire or WirePlumber(if your system is using it). In any case it can not be avoided from our side.

**Reply by foodornt** [reply]
Interesting, I do indeed use them.
I'll keep the discussion open, maybe someone can provide a config script that fixes the issue.
Thank you!

**Comment by servimo** [comment]
Do you have pavucontrol installed? Maybe through this app you can control your volumes.

**Reply by foodornt** [reply]
I do, I can control the volume
It's just annoying that I need to re-adjust volume each time I open EasyEffects
